### PR TITLE
Add a "Contact Us" section to the README

### PR DESCRIPTION
**Pull request by addisoncrump**
This PR adds a section which details how users should contact us with questions or feedback.

### CODE REVIEW

**Comment by addisoncrump** [comment]
I went ahead and cancelled the workflow in CI since there are no CI changes.

File: README.md [hunk 1]
```diff
@@ -140,6 +140,14 @@ If you use LibAFL for your academic work, please cite the following paper:
 }
 ```
 
+## Contact Us
+
+The best way to communicate with us about LibAFL is via [GitHub Discussions](https://github.com/AFLplusplus/LibAFL/discussions).
+This is the best way because it keeps a *searchable record* of conversations about LibAFL, allowing future users to learn from those discussions without needing to ask the same questions again.
+
+We also have a dedicated channel in the [Awesome Fuzzing discord](https://discord.com/invite/QxxTBCw) and have recently created a [Matrix channel](https://matrix.to/#/#libafl:envs.net) as well for more general purpose discussions.
```
Comment (domenukk) [inline README.md:148]: Won't the discord invite be crawled by bots?
And can we bridge matrix now (somehow)?

Comment (addisoncrump) [reply README.md:148]: Yes (it already is) and yes (I'll set this up in the coming days)

**Comment by domenukk** [comment]
People seem to have no issue contacting us as of now, is it okay if we leave this out? But I'm also not havily against having it... It should jsut be closed either way

**Pull request closed without merging**